react/src/components/Table/Table.tsx

```diff
@@ -592,6 +667,18 @@ const Table: React.FC<TableProps> = ({ variantData }) => {
                                 page.map(row => {
                                     prepareRow(row);
                                     const { key, ...restRowProps } = row.getRowProps();
+                                    const caseDetailsCol = headerGroups[0].headers.find(
```

Comment:
Can you make a function called `isCaseDetails` to wrap this logic and move it to the `tableHelpers.ts` file?